### DISCUSSION TITLE
Tooltip does not work for me inside a Modal.Body

### DISCUSSION THREAD

**Discussion by Leekao** [heroui-inc/heroui#1059] [Feedback]
I'm trying to use a Tooltip inside a Modal.Body and it simply does not show, I'm guessing it's a z-index issue.

**Comment by Bhanwra** [comment]
Ran into the same issue. Here's the fix

`#nextui-tooltip, .nextui-tooltip-content {
  z-index: 10000 !important;
}`

NextUI Modal is at zIndex 9999.
